src/webview/WebViewScreen.tsx

```diff
@@ -160,6 +197,14 @@ function WebViewScreen({ route, navigation }: Props) {
           handleSetNavigationTitle(navState.url, navState.title, navState.loading)
         }}
       />
+      {Platform.OS === 'android' && (
+        <WebViewAndroidBottomSheet
+          url={currentUrl}
+          isVisible={showingBottomSheet}
+          onClose={() => toggleBottomSheet()}
+          toggleBottomSheet={toggleBottomSheet}
+        />
```

Comment:
Nothing to do now, but one visual downside of using a BottomSheet inside a screen (vs the BottomSheet being it's own modal screen) is the backdrop doesn't cover the navigation bar of the screen.

Reply:
Made it a modal in https://github.com/valora-inc/wallet/pull/2565/commits/f9388b2a534d370916961651138f0bee72de49d4!